SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/ApplicationLifecycleHelper.cs

```diff
@@ -34,6 +36,11 @@ private static void WinEventHook(IntPtr winEventHookHandle, uint eventType, IntP
                 return;
             }
             var anyNotMinimized = Application.OpenForms.Cast<Form>().Any(form => form.WindowState != FormWindowState.Minimized);
+            if (!started && anyNotMinimized)
+            {
+                started = true;
+                Start?.Invoke();
+            }
             if (suspended && anyNotMinimized)
```

Comment:
Generally speaking nothing will be changed. We don't call resume event on start now.